spec/factories/users.rb

```diff
@@ -0,0 +1,6 @@
+FactoryGirl.define do
+  factory :user do
+    email   "mail@mail.ru"
```

Comment:
Unnecessary spacing detected.<br>Put one space between the method name and the first argument.